docs/app/pages/components/components-sections/select/select/select.component.html

```diff
@@ -58,10 +59,13 @@
                     </div>
                 </div>
                 <div class="row uxd-customize-row">
-                    <div class="col-sm-12">
+                    <div class="col-md-4 col-sm-12">
                         <ux-checkbox [value]="pagingEnabled | async"
                             (valueChange)="pagingEnabled.next($event)">Enable Option Paging</ux-checkbox>
                     </div>
+                    <div class="col-md-4 col-sm-12">
+                        <ux-checkbox [(value)]="readonlyInput">readonlyInput</ux-checkbox>
+                    </div>
```

Comment:
Not to be awkward, but I think I'd rather that this appears next to the other boolean options (multiple, disabled, allowNull).